clang/include/clang/Basic/DiagnosticDriverKinds.td

```diff
@@ -786,4 +786,7 @@ def warn_android_unversioned_fallback : Warning<
   " directories will not be used in Clang 19. Provide a versioned directory"
   " for the target version or lower instead.">,
   InGroup<DiagGroup<"android-unversioned-fallback">>;
+
+def err_android_version_invalid : Error<
```

Comment:
You can name it `err_triple_version_invalid` in case other OSes want to do the same checking.